python/vyos/ifconfig/wireguard.py

```diff
@@ -156,6 +156,63 @@ def show_interface(self):
                 answer += '\n'
         return answer
 
+    def get_latest_handshakes(self):
+
+        """Get latest handshake time for each peer"""
+        output = {}
+
+        # Dump wireguard last handshake
+        _f = self._cmd(f'wg show {self.config["ifname"]} latest-handshakes')
+        # Output:
+        # xxxw=    1732812147
+        # xxx=    0
+        for line in _f.split('\n'):
+            if not line:
+                # Skip empty lines and last line
+                continue
+            items = line.split('\t')
+
+            if len(items) != 2:
+                continue
+
+            output[items[0]] = int(items[1])
+
+        return output
+
+    def reset_peer(self, peer_name=None, public_key=None):
+        from vyos.config import Config
+
+        c = Config()
```

Comment:
Please use this isntead:

```python
from vyos.configquery import ConfigTreeQuery
config = ConfigTreeQuery()
```

It's designed for op-mode.

`return_effective_value` becomes `value` ... just see the implementation of this lightweight class.
`list_effective_nodes` becomes `list_nodes`